src/general/OptionsMenu.cs

```diff
@@ -636,6 +653,40 @@ private void DisplayResolution()
             .FormatSafe(screenResolution.x, screenResolution.y);
     }
 
+    /// <summary>
+    /// Displays the GPU name, the display driver name and used video memory
+    /// </summary>
+    private void DisplayGpuInfo()
+    {
+        var videoAdapterName = VisualServer.GetVideoAdapterName();
+        gpuName.Text = TranslationServer.Translate("AUTO_GPU_NAME")
+            .FormatSafe(videoAdapterName);
+
+        if (OS.GetCurrentVideoDriver() == OS.VideoDriver.Gles2)
+        {
+            // Gles2 is being used
+            displayDriverName.Text = TranslationServer.Translate("AUTO_DISPLAY_DRIVER_NAME")
```

Comment:
The launcher actually allows forcing the renderer to be gles2 so I wouldn't use "AUTO" in the name here.